src/services/loginService.js

```diff
@@ -0,0 +1,29 @@
+import { url } from './api';
+
+export const LoginService = {
+  login: async (email, password) => {
+    try {
+      const response = await fetch(`${url}/login`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ email, password }),
+      });
+
+      if (!response.ok) throw new Error('Erro ao fazer o login');
+
+      const data = await response.json();
+      return {
+        success: true,
+        token: data.token,
```

Comment:
aqui não precisaria desse await, ou como ta retornando só o token vc pode fazer uma desestruturação, seria algo do tipo 
```suggestion
      const { token } = await response.json();
      return {
        success: true,
        token: token,
```

ai precisaria do await